code/datums/components/burning.dm

```diff
@@ -23,27 +23,29 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 		return
 	src.fire_overlay = fire_overlay
 	if(fire_particles)
-		particle_effect = new(atom_parent, fire_particles)
+		// burning particles when they stack on mobs, so do not do that
+		particle_effect = new(atom_parent, fire_particles, isitem(atom_parent) ? NONE : PARTICLE_ATTACH_MOB)
 	atom_parent.resistance_flags |= ON_FIRE
 	START_PROCESSING(SSfire_burning, src)
 
 /datum/component/burning/RegisterWithParent()
 	. = ..()
+	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, PROC_REF(on_extinguish))
```

Comment:
```suggestion
	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, PROC_REF(on_examine))
```
this is soooooooo not what you meant player

Reply:
sorry im a gemini